pkg/virt-controller/watch/vmi.go

```diff
@@ -1855,6 +1909,10 @@ func (c *VMIController) updateVolumeStatus(vmi *virtv1.VirtualMachineInstance, v
 					Capacity:     pvc.Status.Capacity,
 					Preallocated: kubevirttypes.IsPreallocated(pvc.ObjectMeta.Annotations),
 				}
+				possibleGuestSize, ok := c.possibleGuestSize(pvc)
+				if ok && c.clusterConfig.ExpandDisksEnabled() {
+					status.PersistentVolumeClaimInfo.PossibleGuestSize = &possibleGuestSize
+				}
```

Comment:
I know the term `possibleGuestSize` is ambiguous on purpose because it's supposed to mean "this is what the guest could use".

Could we make this all a bit less ambiguous by storing the `filesystemOverhead` on the vmi's status instead?